doc/bash_completion.d/yunohost

```diff
@@ -0,0 +1,125 @@
+#
+# completion for yunohost
+# automatically generated from the actionsmap
+#
```

Comment:
This file is not meant to be commited é_è

Reply:
should we add it inside .gitignore ?

Reply:
sure